integration/netmon/netmon_test.bats

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+# Environment variables
+IMAGE="busybox"
+NETWORK_NAME="test"
+CONTAINER_NAME="containerA"
+PAYLOAD_ARGS="tail -f /dev/null"
+
+setup() {
+	clean_env
+
+	# Check that processes are not running
+	run check_processes
+	echo "$output"
+	[ "$status" -eq 0 ]
+
+	extract_kata_env
+
+	# Enabling netmon at the configuration file
+	sed -i 's|#enable_netmon = true|enable_netmon = true|' ${RUNTIME_CONFIG_PATH}
+}
+
+@test "test netmon" {
+	docker network create $NETWORK_NAME
+	docker run --runtime=$RUNTIME -d --name $CONTAINER_NAME $IMAGE sh -c $PAYLOAD_ARGS
+	docker network connect $NETWORK_NAME $CONTAINER_NAME
+	# Get ip address
+	run docker inspect --format "{{.NetworkSettings.IPAddress}}" $CONTAINER_NAME
```

Comment:
Ok so this does not retrieve the right information, and it does check the IP address either.

I would suggest that we test the addition of the extra network worked by:
- `docker exec $CONTAINER_NAME ifconfig` or `docker exec $CONTAINER_NAME ip a` before we add the new network, this way, we know about what was there as a reference.
- `docker exec $CONTAINER_NAME ifconfig` or `docker exec $CONTAINER_NAME ip a` after the new network has been added. By comparing to the reference, we should find that a new interface has been added.
- Check the IP address of the new interface, and compare it to the IP of the network created
- Now remove the network by using `docker disconnect`
- Check `docker exec $CONTAINER_NAME ifconfig` or `docker exec $CONTAINER_NAME ip a` again, to verify you got back to the initial reference.

This would check that the network being added is properly added and shows up inside the container. This would also make sure the removal works properly.
And last thing we could add, would be to check the connectivity, by maybe pinging the bridge on the host.

Reply:
@sboeuf thanks for the feedback, let me work on this

Reply:
@sboeuf changes applied :)